server/test/utils.ts

```diff
@@ -51,20 +51,25 @@ function makeUser({
   };
 }
 
-async function createUser(options: Partial<UserObject> = {}) {
-  const userObject = makeUser(options);
+async function createSavedUserModel(options: Partial<UserObject> = {}) {
+  const userObject = makeUnsavedUserModel(options);
   // insert into DB after creation
   return await User.create(userObject);
 }
 
+function createManyUsers(numberOfUsers: number): UserObject[] {
+  return Array.from({ length: numberOfUsers }, makeUnsavedUserModel);
+}
+
 function getExpectedUserObject(user: User) {
   const allowedFields = User.allowedFields;
   const pickedUser = _.pick(user, allowedFields);
   return pickedUser as User;
 }
 
 export {
-  makeUser,
-  createUser,
+  makeUnsavedUserModel,
+  createManyUsers,
+  createSavedUserModel,
```

Comment:
Careful when using the word Model as that implies we're returning an actual User Model. `makeUnsavedUserModel` is just returning a plain object with properties that match our User fields. It's not returning an actual Sequelize model (instance of `server/models/User.model.ts`).

Perhaps keeping `createUser` the same, as I think that's pretty straightforward... but then being more verbose on the others? eg. `makeUserObject`/`makeUserObjects` or `generateUserObject`/`generateUserObjects`?

Reply:
alright, I'll change that up